tools/rules/trailing_whitespace

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Why have you changed this?

Reply:
Apologies and well-spotted, it crept in during debugging. Sorted now.